app/controllers/food_controller.rb

```diff
@@ -0,0 +1,31 @@
+class FoodController < ApplicationController
+  def index
+    @foods = Food.all
+  end
+
+  def new
+    @food = Food.new
+  end
+
+  def create
+    @food = Food.new(food_params)
+    if @food.save
+      redirect_to new_food_path, notice: 'Food has been added.'
+    else
+      flash.now[:alert] = 'Error! Unable to add food.'
+      render :show
+    end
+  end
```

Comment:
I checked again and I can add food from my system without any problem. Could you please elaborate the issue

Reply:
When I tried creating the food I got the error above. If you think the error is from my side and everything is fine you can ignore this and work on the other requested changes. However, it would be nice to maybe create a seed data so that it may be easier for the next reviewer to populate the data into the app without encountering such issues. :pray: